src/components/Posts.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class Posts extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            posts: []
+        };
+    }
+    componentWillMount() {
```

Comment:
This is how `Life-Cycle-Methods` are behaving.
![image](https://user-images.githubusercontent.com/15275353/39672991-f47f9640-5151-11e8-93df-8efe0a36d824.png)

![image](https://user-images.githubusercontent.com/15275353/39672993-fd5a4378-5151-11e8-8643-705e9a1df047.png)